examples/pytorch/question-answering/run_qa.py

```diff
@@ -89,6 +175,21 @@ class DataTrainingArguments:
     Arguments pertaining to what data we are going to input our model for training and eval.
     """
 
+    ####################################################################################
+    # Start SparseML Integration
+    ####################################################################################
+    nm_prune_config: Optional[str] = field(
+        default="recipes/noprune1epoch.yaml",
+        metadata={"help": "The input file name for the Neural Magic pruning config"},
+    )
+    do_onnx_export: bool = field(default=True, metadata={"help": "Export model to onnx"})
```

Comment:
let's remove this parameter and overload the onnx_export_path, if that is set, then we'll export to onnx